gitter/preprocessing.py

```diff
@@ -0,0 +1,43 @@
+import nltk
+
+
+class Splitter(object):
+    def __init__(self):
+        self.nltk_splitter = nltk.data.load('tokenizers/punkt/english.pickle')
+        self.nltk_tokenizer = nltk.tokenize.TreebankWordTokenizer()
+
+    def split(self, text):
+        """
+        input format: a paragraph of text
+        output format: a list of lists of words.
+        e.g.: [['this', 'is', 'a', 'sentence'],
+              ['this', 'is', 'another', 'one']]
+        """
+        sentences = self.nltk_splitter.tokenize(text)
```

Comment:
I'm not familiar with `nltk`, but I wonder what would happen if the text is not in English? Would it be an issue?